tests/unit/test_configuration.py

```diff
@@ -21,10 +21,29 @@ def user_configuration_yaml(tmp_path):
     return user_config_path
 
 
-def test_get_configuration(user_configuration_yaml):
-    config = get_configuration()
-    assert config
-    assert isinstance(config, ConfigTree)
+def test_get_configuration(mocker):
+    """Tests that the default configuration can be retrieved."""
+    mock = mocker.patch("pseudopeople.utilities.ConfigTree")
```

Comment:
Is it worth checking that the returned object is an instance of `vivarium.ConfigTree` to make sure that `pseudopeople.utilities.ConfigTree` doesn't happen to be some different type?

Reply:
It's not possible AFAICT. Because of the mocking, iIt becomes a mock object. It isn't a ConfigTree.